src/pages/docs/golem/overview/testnet-mainnet.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Golem Testnet and Mainnet explained.
+description: Introduction to Golem Network networks.
+type: Article
+---
+
+# Testnet / Mainnet, Addresses and Wallets
+
+## Networks
+
+`Testnet` is the part of the Golem Network, that accepts payments on test blockchains. By default, the requestor node uses `testnet`.
+
+Requestor users can select their preferred blockchain. Their demand will be matched by offers from providers that accept payments on test blockchains. Golem Network and its community operate some providers on the `testnet` that serves as a testing environment for developers.
+
+`yagna payment fund` is a command to obtain tokens necessary to run applications on the `testnet`: both tGLM and the chain native token used to pay gas fees.
+
```

Comment:
We should add in this section that Mainnet tokens (Polygon) can be obtained by running `yagna payment fund --network=polygon` in Yagna CLI too.

Reply:
added